morphir/runtime/src/org/finos/morphir/runtime/TypedMorphirRuntime.scala

```diff
@@ -13,7 +13,10 @@ import org.finos.morphir.datamodel.*
 import org.finos.morphir.runtime.environment.MorphirEnv
 import org.finos.morphir.runtime.exports.*
 
-trait TypedMorphirRuntime extends MorphirRuntime[scala.Unit, UType] {
+trait TypedMorphirRuntime extends MorphirRuntime {
+  type TypeAttribs  = TypedMorphirRuntime.TypeAttribs
+  type ValueAttribs = TypedMorphirRuntime.ValueAttribs
+
   final def evaluate(
       entryPoint: Value[scala.Unit, UType],
```

Comment:
This should be using the type members (TypeAttribs and ValueAttribs)

Reply:
Makes sense! Will change